wave_matching/include/wave/matching/ndt.hpp

```diff
@@ -0,0 +1,29 @@
+// Class to wrap PCL ICP matcher
+
+#ifndef WAVE_MATCHING_NDT_HPP
+#define WAVE_MATCHING_NDT_HPP
+
+#include <pcl/point_types.h>
+#include <pcl/point_cloud.h>
+#include <pcl/registration/ndt.h>
+#include "wave/matching/matcher.hpp"
+#include "wave/matching/pcl_common.hpp"
+
+namespace wave {
+
+class NDTMatcher : public Matcher<PCLPointCloud> {
+ public:
+    explicit NDTMatcher(float resolution, const std::string &config_path);
+    void setRef(const PCLPointCloud &ref);
+    void setTarget(const PCLPointCloud &target);
+    bool match();
+
+ private:
+    pcl::NormalDistributionsTransform<pcl::PointXYZ, pcl::PointXYZ> ndt;
+    PCLPointCloud ref, target, final;
+    const float min_res = 0.05f;
```

Comment:
In this case the "f" makes sense because its explicity saying this float is of type `float`, without it by default its a `double` and if you do `float min_res = 0.05` you are implicitly casting the double to float, small detail but this case it makes perfect sense to have the "f" at the end.